### DISCUSSION TITLE
Set Desired Stance doesn't always seem to work when using custom stances?

### DISCUSSION THREAD

**Discussion by xkamronx** [Sixze/ALS-Refactored#504] [Q&A]
I added some custom stance gameplay tags where required, then added a 'force' for the tag to be set from another post here that said it had to do so to make it function to begin with.

When I call Set Desired Stance (it's currently being set on a RepNotify for driving, for example) it doesn't always seem to update across the network, and thus the character isn't in the driving animation on other clients.

```
Void AAlsCharacter::ApplyDesiredStance()
{
	if (!LocomotionAction.IsValid())
	{

		Stance = DesiredStance;     // < Added this line to force custom stances

		if (LocomotionMode == AlsLocomotionModeTags::Grounded)
```

If I'm calling this setup wrong to change the character's stance with Set Desired Stance, what is the preferred method for actually changing the character stance to add other actions / states to ALS Refactored?

**Comment by xkamronx** [comment]
Some additional testing and searching. When trying to create a new Stance, there's a bit more needed to make it work.

By default, new stances aren't replicated either since Crouch and Uncrouch are what actually change the stance, so calling Set Desired Stance with a custom Stance won't work in multiplayer at all.

I added a simple switch on gameplay tag in the animation graph for simple testing of "sitting down" that just loops an animation of sitting down if the character is in the new Stance of "Sitting", but it doesn't replicate without C++ modification.

Getting the character back to Standing from a custom Stance is also somewhat tricky for multiplayer, as Set Desired Stance to Standing doesn't pull them out of the custom state either.

**Comment by Jinkel92** [comment]
It seems custom tag abilities doesn't work.
